scalableminds/webknossos
# change action button label if annotation does not belong to user #2384

### Mailable description of changes:
- [Admin.Tasks] Relabeled button action to "view" for annotations for which one is not the owner.

![image](https://user-images.githubusercontent.com/1105056/36983770-83c517a2-2093-11e8-9d3a-d585849575b8.png)


### Steps to test:
- Get a task with first user from dashboard. 
- Go to Admin --> Tasks. Find the task, expand the subtable. Click on "Actions". The first entry should say "trace" because you are the owner of that annotation
- Create a second user. Get a task for him.
- As the first user, check the task subtable again. Click on "Actions". The first entry should say "view" because you are NOT the owner of that annotation.

### Issues:
- fixes #2384

------
